docker/bin/build_staticfiles.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -exo pipefail
+
+rm -rf ./static
+
+python birdbox/manage.py collectstatic --noinput
```

Comment:
Can you use `--clear` here to replace the above `rm -rf` line?

Reply:
Maybe, yeah. I basically stole this wholesale from Bedrock. Will take a look